packages/algob/sample-project/test/sample-test.js

```diff
@@ -1,5 +1,5 @@
-import { getProgram } from '@algo-builder/algob';
-import { Runtime, StoreAccount, types } from '@algo-builder/runtime';
+const { getProgram } =  require('@algo-builder/algob');
+const { Runtime, StoreAccount, types } = require('@algo-builder/runtime');
```

Comment:
why we need to use `require`?

Reply:
We are adding JS files to mocha to run tests [https://github.com/scale-it/algo-builder/blob/master/packages/algob/src/builtin-tasks/test.ts#L15](https://github.com/scale-it/algo-builder/blob/master/packages/algob/src/builtin-tasks/test.ts#L15).

With `import` it was showing: `Error Occurred: Cannot use import statement outside a ..` with `algob test`. So i changed to "import from" to "require"